services/workflows-service/entrypoint.sh

```
@@ -0,0 +1,70 @@
+    dep_tool="ecs-fargate"
+  fi
+}
+
+## Main Block
+get_cloud_provider
+get_tool
+get_hostname
+check_for_fargate
+get_current_time
+
+infra_json='{"cloudProvider":"'"$cloud_provider"'","tool":"'"$dep_tool"'","hostname":"'"$hostname"'", "currentTime": "'"$currentTime"'"}'
```

Shellcheck

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)